### PR TITLE
refactor: MCDU model behaviors into src

### PR DESCRIPTION
**Pull request by theomessin**

### CODE REVIEW

File: src/behavior/src/A32NX_Interior_MCDU.xml [hunk 1]
```diff
@@ -0,0 +1,391 @@
...
+			<WWISE_EVENT_2>fmc_push_button_off</WWISE_EVENT_2>
+            <NODE_ID_BRT_DIM>PUSH_MCDU#SIDE#_BRT</NODE_ID_BRT_DIM>
+			<ANIM_NAME_BRT_DIM>PUSH_MCDU#SIDE#_BRT</ANIM_NAME_BRT_DIM>
+            <NODE_ID_BRT_DIM_SEQ1>PUSH_MCDU#SIDE#_BRT_SEQ1</NODE_ID_BRT_DIM_SEQ1>
+            <NODE_ID_BRT_DIM_SEQ2>PUSH_MCDU#SIDE#_DIM_SEQ1</NODE_ID_BRT_DIM_SEQ2>
+		</DefaultTemplateParameters>
+
+        <Update Once="True">
+            0.5 (&gt;L:A32NX_MCDU_#SIDE#_BRIGHTNESS)
+        </Update>
```
Comment (theomessin) [inline src/behavior/src/A32NX_Interior_MCDU.xml:340]: Too much magic going on here...

**Comment by theomessin** [comment]
@davidwalschots come to think of it, this warrants its own PR so as not to have huge PRs for master 😄 shouldn't take too long to get it merged

**Comment by davidwalschots** [comment]
@theomessin you might want to actually target `master` on the `a32nx` repo then? Otherwise it will become part of my PR ;-).

**Comment by davidwalschots** [comment]
@theomessin Nevermind. You closed this PR. I got confused. Will you make a PR on the non-forked repo?

**Comment by theomessin** [comment]
https://github.com/flybywiresim/a32nx/pull/4045

**Pull request closed without merging**